css/main.css

```diff
@@ -0,0 +1,73 @@
+:root {
+  --soft-blue: hsl(215, 51%, 70%);
+  --cyan: hsl(178, 100%, 50%);
+  background-color: hsl(217, 54%, 11%);
+
+  --card-bg-color: hsl(216, 50%, 16%);
+  --dark-blue-line-color: hsl(215, 32%, 27%);
+}
+
+* {
+  font-family: "Poppins", sans-serif;
+}
+body {
+  width: 100%;
+}
+.main-body {
+  width: 100%;
+}
+.trivia-cards-container {
+  display: flex;
+  flex-direction: row;
```

Comment:
you don't need to specify this. As the default direction is row.

Reply:
Sure